kratos/python/add_strategies_to_python.cpp

```diff
@@ -387,6 +387,7 @@ namespace Kratos
                 .def("BuildAndSolve", &BuilderAndSolverType::BuildAndSolve)
                 .def("BuildRHSAndSolve", &BuilderAndSolverType::BuildRHSAndSolve)
                 .def("ApplyDirichletConditions", &BuilderAndSolverType::ApplyDirichletConditions)
+                .def("ApplyConstraints", &BuilderAndSolverType::ApplyConstraints)
```

Comment:
This changes should be in a independent PR..m the rest is OK

Reply:
yes please make a PR for the changes in the Core